submodules/externals/x264

```diff
@@ -1 +1 @@
-Subproject commit 1840d2a6de0e832d5c770e0810bcaf5c5ca7a7ff
+Subproject commit 3f516c5238d0c536ea03c8e5334d231facf9f31b
```

Comment:
Looking at https://github.com/BelledonneCommunications/linphone-iphone, this is 1840d2a6de0e832d5c770e0810bcaf5c5ca7a7ff, why would I want to use 3f516c5238d0c536ea03c8e5334d231facf9f31b?